src/index.js

```diff
@@ -221,3 +256,4 @@ function joinClasses(...classes) {
 //   globalObject.glamorous = glamorous
 // }
 export default glamorous
+export {ThemeProvider}
```

Comment:
You can feel free to delete the commented out code above this. That was a mistake.

Also, as discussed, I think that if you add: `Object.assign(glamorous, {ThemeProvider})` that should make our export be what we want.